public/controllers/management/components/management/cdblists/components/columns.tsx

```diff
@@ -0,0 +1,182 @@
+                    const result = await resourcesHandler.getFileContent(item.filename);
+                    const file = { name: item.filename, content: result, path: item.relative_dirname };
+                    this.props.updateListContent(file);
+                  } catch (error) {
+                    const options: UIErrorLog = getErrorOptions(
+                      error,
+                      'Lists.editFileContent'
+                    );
+                    getErrorOrchestrator().handleError(options);
+                  }
+                }}
+                color="primary"
+              />
+              <WzButtonPermissionsModalConfirm
+                buttonType="icon"
+                permissions={getDeleteButtonPermissions(item)}
+                tooltip={{ position: 'top', content: (defaultItems.indexOf(`${item.relative_dirname}`) === -1) ? `Delete ${item.filename}` : `The ${item.filename} list cannot be deleted` }}
+                aria-label="Delete file"
+                iconType="trash"
+                isDisabled={defaultItems.indexOf(`${item.relative_dirname}`) !== -1}
+                onConfirm={async () => {
+                  try {
+                    this.props.removeItems([item])
+                  } catch (error) {
+                    const options: UIErrorLog = getErrorOptions(
+                      error,
+                      'Lists.deleteFile'
+                    );
+                    getErrorOrchestrator().handleError(options);
+                  }
+                }}
+                onClose={async (ev) => ev.stopPropagation()}
+                onClick={(ev) => ev.stopPropagation()}
+                color="danger"
+                modalTitle={'Are you sure?'}
+                modalProps={{
+                  buttonColor: 'danger',
+                  onClick: (ev) => ev.stopPropagation()
+                }}
+              />
+              <WzButtonPermissions
+                buttonType='icon'
+                permissions={getReadButtonPermissions(item)}
+                aria-label="Export list"
+                iconType="exportAction"
+                tooltip={{ position: 'top', content: `Export ${item.filename} content` }}
+                onClick={async (ev) => {
+                  try {
+                    ev.stopPropagation();
+                    await exportCsv(`/lists`,
+                      [{ _isCDBList: true, name: 'filename', value: `${item.filename}` }],
+                      item.filename
+                    )
+                  } catch (error) {
+                    const options: UIErrorLog = getErrorOptions(
+                      error,
+                      'Lists.exportFile'
+                    );
+                    getErrorOrchestrator().handleError(options);
+                  }
+                }}
+                color="primary"
+              />
+            </div>
+          )
+        }
+      }
```

Comment:
I think this inner function could be moved outside the constructor, as a class method.